web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -126,7 +124,7 @@ describe("<LicenseTask />", () => {
     it("starts on application tab when lastUpdatedISO is empty and visits status tab by clicking secondary button", () => {
       useMockBusiness({
         licenseData: generateLicenseData(
-          {},
+          { lastUpdatedISO: undefined },
```

Comment:
boulder - there are multiple instances of lastUpdatedISO, due to this change, the test is passing for the wrong reason.